go-chaos/worker/chaos_worker.go

```diff
@@ -197,7 +221,7 @@ func getTargetClusterVersion(namespace string) string {
 	port, closeFn := k8Client.MustGatewayPortForward(0, 26500)
 	defer closeFn()
 
-	zbClient, err := internal.CreateZeebeClient(port)
+	zbClient, err := internal.CreateZeebeClient(port, credentials)
```

Comment:
❓ 🔧 We could also pass in here nil and then remove the parameter of the `getTargetClusterVersion` function